wrappers/python/examples/llama_training.py

```diff
@@ -103,114 +94,194 @@
+    train_tokens_raw[:num_train_tokens_truncated],
+    order='F', dtype=np.int64)
+train_tokens = train_tokens_trunc.reshape(num_train_batches,
+                                    num_minibatch,
+                                    args.minibatch_size,
+                                    args.seq_len + 1)
+
+time0 = time.time()
+batch_input = []
+batch_output = []
+x_traits = nntile.tensor.TensorTraits(
+        [args.seq_len, args.minibatch_size],
+        [args.seq_len_tile, args.minibatch_size_tile])
+x_distr = [0] * x_traits.grid.nelems
+for i in range(num_train_batches):
+    minibatch_input = []
+    minibatch_output = []
+    for j in range(num_minibatch):
+        x = nntile.tensor.Tensor_int64(x_traits, x_distr, next_tag)
+        next_tag = x.next_tag
+        x.from_array(np.asfortranarray(train_tokens[i, j, :, :-1].T))
+        minibatch_input.append(x)
+        y = nntile.tensor.Tensor_int64(x_traits, x_distr, next_tag)
+        next_tag = y.next_tag
+        y.from_array(np.asfortranarray(train_tokens[i, j, :, 1:].T))
+        minibatch_output.append(y)
+    batch_input.append(minibatch_input)
+    batch_output.append(minibatch_output)
+time1 = time.time() - time0
+print("From PyTorch loader to NNTile batches in {} seconds".format(time1))
+# Set up learning rate and optimizer for training
+if args.optimizer == "adam":
+    optimizer = nntile.optimizer.Adam(llama_nntile.get_parameters(),
+            args.lr, next_tag)
+elif args.optimizer == "adamw":
+    optimizer = nntile.optimizer.AdamW(llama_nntile.get_parameters(),
+            args.lr, next_tag)
+elif args.optimizer == "sgd":
+    optimizer = nntile.optimizer.SGD(llama_nntile.get_parameters(),
+            args.lr, next_tag)
+next_tag = optimizer.get_next_tag()
+# Define Cross Entropy loss function
+loss, next_tag = nntile.loss.CrossEntropy.generate_simple(
+        llama_nntile.activations[-1], next_tag,
+        scale=1.0 / (args.batch_size * args.seq_len))
+# Set up training pipeline
+pipeline = nntile.pipeline.Pipeline(batch_input, batch_output,
+        llama_nntile, optimizer, loss, args.nepochs)
+# Warmup training
+# nntile.starpu.pause()
+nntile.starpu.profiling_enable()
+pipeline.train_async()
+# nntile.starpu.resume()
+nntile.starpu.wait_for_all()
+nntile.starpu.profiling_disable()
+time1 = time.time() - time0
+print("NNTile training time: {} seconds".format(time1))
+print("NNTile training throughput tokens/sec: {}".format(
+        args.nepochs * num_train_batches * args.batch_size
+        * args.seq_len / time1))
+nflops_fwd_seq = llama_nntile.get_model_forward_flops()
+nflops_bwd_seq = llama_nntile.get_model_backward_flops()
+nflops_seq = nflops_fwd_seq + nflops_bwd_seq
+print("Fwd flops={}".format(nflops_fwd_seq))
+print("Bwd flops={}".format(nflops_bwd_seq))
+print("Num train batches={}".format(num_train_batches))
+print("Time={}".format(time1))
+print("NNTile performance: {} Tflops/s".format(nflops_seq
+        * args.nepochs * num_train_batches
```

Comment:
multiply by `num_minibatches` because each epoch contains given number of train batches, each batch contains `minibatch` of mini batches and number of flops is given per a mini batch.